### DISCUSSION TITLE
LLM settings in UI

### DISCUSSION THREAD

**Discussion by trzonne** [strnad/CrewAI-Studio#21] [Ideas]
What a great find CrewAI-Studio is! Love it!

I wonder if it is possible to incorporate the LLM configs / in the UI also, so I can add Perplexity, Ollama and such. Or better, incorporate those 2 in the package.
Perhaps management of API keys also?

**Comment by cpretzinger** [comment]
Anybody been working on this one? I have used / attempted to build almost all of these - and this is ththe only one I had "working " until I broke it
